lib/matplotlib/tests/test_collections.py

```diff
@@ -325,6 +325,18 @@ def test__EventCollection__set_linestyle():
     splt.set_title('EventCollection: set_linestyle')
 
 
+@image_comparison(baseline_images=['EventCollection_plot__set_ls_dash'])
```

Comment:
@tacaswell Thanks for the reminder. I havn't touched mpl code for a while now. I'll check the Changelog more regulary.

The test is corrected now.